car_dreamer/carla_wpt_env.py

```diff
@@ -131,7 +131,8 @@ def reward(self):
         return total_reward, info
 
     def is_destination_reached(self):
-        return len(self.waypoints) <= 3
+        # return len(self.waypoints) <= 3
```

Comment:
wpt env is a base class that supports multiple different tasks like follow, right turn, left turn, and more. wondering if the change does not affect other tasks